src/webapp/src/App.tsx

```diff
@@ -1,22 +1,31 @@
 import React from 'react';
 import { Button, GlobalStyle } from '@small-ads/ui';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 import { Characters } from './characters/Characters';
 import { Person } from './person/Person';
 import { Auth } from './auth/Auth';
 import { Navbar } from './Navbar';
+import { Login } from './auth/Login';
 
 export const App = () => (
   <>
-    <Navbar />
-    <GlobalStyle />
-    <Auth />
-    <Person />
-    <Button onClick={() => {}} type='secondary'>
-      Invite Team
-    </Button>
-    <Button onClick={() => {}} type='primary'>
-      New Project
-    </Button>
-    <Characters />
+    <Router>
+      <GlobalStyle />
+      <Navbar />
+      <Switch>
+        <Route path='/login' component={Login} />
+      </Switch>
+      <Route exact path='/'>
+        <Auth />
+        <Person />
+        <Button onClick={() => {}} type='secondary'>
+          Invite Team
+        </Button>
+        <Button onClick={() => {}} type='primary'>
+          New Project
+        </Button>
+        <Characters />
```

Comment:
Can we get rid of those in this PR please?

Reply:
I was hoping for you to say that! I hate there! :D

Reply:
I should only remove them from the UI, right? 

Or do you want me to completely remove the components from the project as well?

Reply:
If you think we might need them again lets keep them. But I doubt it. However feel free to remove them in another PR.

Reply:
Yes, I will do it in another PR better.